node_modules/node-red-node-email/61-email.html

```diff
@@ -84,7 +88,8 @@
             dname: {value:""},
             to: {required: true},
             from: {required: true},
-            subject: {required: true}
+            subject: {required: true},
+            body : {required: true}
```

Comment:
I'm a bit lost here... 
I was under the impression that the email body would come from a flow variable set somewhere.
If we are to put the email body here, it would have to work as a template node right? That being the case, we might want to look into having ACE configured on this field, allowing syntax highlighting to be performed here.

Anyways, I'm under the impression that the body field shouldn't be required, thus allowing the user the possibility of using the same body definition for multiple outputs.